src/components/Tokens/TokenTable/TokenRow.tsx

```diff
@@ -453,6 +453,9 @@ export const LoadedRow = forwardRef((props: LoadedRowProps, ref: ForwardedRef<HT
     search_token_address_input: filterString,
   }
 
+  // A simple 0 price indicates the price is not currently available from the api
+  const price = token.market?.price?.value === 0 ? '-' : formatUSDPrice(token.market?.price?.value)
```

Comment:
do you want to show '-' if it's undefined as well?

Reply:
formatUSDPrice will do that already, 0 is the only special case we care about here